ironic_prometheus_exporter/parsers/ipmi.py

```diff
@@ -26,6 +26,10 @@ def metric_names(payload, prefix, sufix, **kwargs):
             e = re.sub(r'[\d].*$', '', entry.lower())
             e = re.sub(r'[\(\)]', '', e).split()
             label = '_'.join(e)
+        elif special_label == 'voltage':
+            e = re.sub(r'([\d+]v)|([\d+].[\d*]v)', '', entry.lower())
```

Comment:
I have no way to evaluate whether this regex is doing the right thing, because there's no sample data here. It would really help readers of the code understand what's going on if you add a sample comment with real data in each section where parsing is being done.

Reply:
ack